client/src/app/pages/issues/affected-applications/affected-applications.tsx

```diff
@@ -181,7 +184,34 @@ export const AffectedApplications: React.FC = () => {
                     <Th {...getThProps({ columnKey: "name" })} />
                     <Th {...getThProps({ columnKey: "description" })} />
                     <Th {...getThProps({ columnKey: "businessService" })} />
-                    <Th {...getThProps({ columnKey: "effort" })} />
+                    <Th
+                      {...getThProps({ columnKey: "effort" })}
+                      style={{ paddingRight: 0 }}
+                    >
+                      <div style={{ display: "flex", alignItems: "center" }}>
```

Comment:
Typically we would like to use the PF design system for flexbox here. See the implementation in "IconedStatus" for an example.

Reply:
Thanks @ibolton336  has made some modifications. Please review and inform me if any further adjustments are needed.